docs/pages/eas-update/runtime-versions.mdx

```diff
@@ -118,6 +118,26 @@ It's important to know that this policy does require the developer to manage the
 
 Also, if you select a different native version between iOS and Android, you'll end up with builds and updates with separate runtime versions. When you publish an update, EAS CLI will detect that two updates are needed, and it will create two updates with the required runtime versions and publish them separately.
 
+### `"fingerprint"` runtime version policy (experimental)
+
+We provide the `"fingerprint"` runtime version policy for a project that has custom native code that may change between builds:
+
+```json
+{
+  "expo": {
+    "runtimeVersion": {
+      "policy": "fingerprint"
+    }
+  }
+}
+```
+
+This policy is well-suited for projects incorporating custom native code that require safe updates without native conflicts. Unlike the "nativeVersion" policy, the fingerprint option automatically scans your project's native dependencies to update the runtimeVersion accordingly.
+
+Please note that enabling fingerprinting may lengthen build times, as the fingerprinting process can be resource-intensive.
+
+Attention: Use of the experimental fingerprint runtime policy may result in unexpected system behavior. This policy is still under active development as part of the expo-fingerprint beta package, and its specifications may change in the future.
```

Comment:
```suggestion
> **warning** Using the experimental fingerprint runtime policy may result in unexpected system behavior. This policy is still under active development as part of the `expo-fingerprint` beta package, and its specifications may change in the future.
```

If this needs to be a note or a warning, let's use Blockquote (`>`) or a warning callout (`> **warning**`). I personally think since this is in beta, we can add a warning callout since that's the pattern we follow in Expo docs.